<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Appearance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="LightThemeBtn">
              <property name="text">
               <string>Light Theme</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="DarkThemeBtn">
              <property name="text">
               <string>Dark Theme</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Recordings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Recording file format</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="saveFormatComboBox"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Save records to</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="saveRootTextEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="SelectDataDirBtn">
                 <property name="text">
                  <string>Browse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="resetDefaultBtn">
           <property name="maximumSize">
            <size>
             <width>256</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Reset Default</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
